docs/_guide/styles.md

```diff
@@ -238,7 +294,23 @@ h1 {
 }
 ```
 
-## Inherited properties
+#### Styling slotted elements {#slotted}
+
+Use the `::slotted()` CSS pseudo-element to select light DOM elements that have been included in shadow DOM via the `<slot>` element.
```

Comment:
```suggestion
Use the `::slotted()` CSS pseudo-element to select light DOM children that have been included in shadow DOM via the `<slot>` element.
```

"children" because I suspect it's a bit more inline with intuition here. If we can explain or link to projection somewhere (maybe in the "what you can style" part), that might help.